package.json

```diff
@@ -28,10 +28,14 @@
     "lint:worker": "gulp lint:worker",
     "format:worker": "gulp format:worker",
     "test": "npm run test:node && npm run test:worker",
-    "test:node": "gulp build && jest",
-    "test:worker": "gulp test:worker",
-    "coverage:node": "gulp build && jest --coverage && open-cli coverage/lcov-report/index.html",
-    "postinstall": "gulp build"
+    "test:node": "npm run postinstall && jest",
```

Comment:
Why this new "npm run postinstall" here? I prefer to leave it as it was, without referencing another "script":

```json
"test:node": "make -C worker && jest",
```

Reply:
The same in "coverage:node" script. Let's use "make -C worker && jest ..." instead of "npm run postinstall && jest ...".

Reply:
Ah ok, I see. If we use "make -C worker" then it won't work in Windows...

Reply:
Ok, let's use "npm run postinstall" then.